.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    "configurations": [
+        {
+            "name": "run audacity",
+            "type": "cppvsdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build/install/bin/audacity.exe",
+            "preLaunchTask": "build audacity",
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "visualizerFile": "${workspaceFolder}/qt6.natvis",
+            "sourceFileMap": {
+                "C:/Users/qt/work/qt": "${QTDIR}/../Src"
+            },
+            "windows": {
+                "environment": [
+                    {
+                        "name": "PATH",
+                        "value": "${env:PATH};${command:qt-cpp.kitDirectory}${/}bin"
```

Comment:
It is better to simply write in the instructions (and when trying to configure it, it is indicated) that us need to add Qt to the PATH